utils/tokenGenerator.js

```diff
@@ -0,0 +1,17 @@
+import jwt from "jsonwebtoken";
```

Comment:
utils is not appropriate place to work with database. Repository layer should be added, or at least service layer